src/CSF.SRDashboard.Client/PageValidators/SearchErrorObj.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.PageValidators
+{
+    public enum errorType
```

Comment:
Capitalize the first word of an Enum like a Class.
```suggestion
    public enum ErrorType
```